project-base/docker/conf/docker-compose-mac.yml.dist

```diff
@@ -24,14 +24,18 @@ services:
 
     php-fpm:
         build:
-            context: docker/php-fpm
+            context: project-base/docker/php-fpm
             args:
                 www_data_uid: 501
                 www_data_gid: 20
         container_name: shopsys-framework-php-fpm
+        environment:
+            - COMPOSER_AUTH={"github-oauth":{"github.com":"place-your-token-here"}}
         volumes:
             - shopsys-framework-sync:/var/www/html
-            - ./docker/php-fpm/php-ini-overrides.ini:/usr/local/etc/php/php.ini
+            - shopsys-framework-vendor-sync:/var/www/html/vendor
+            - shopsys-framework-webserver-sync:/var/www/html/project-base/web
+            - ./project-base/docker/php-fpm/php-ini-overrides.ini:/usr/local/etc/php/php.ini:delegated
```

Comment:
I think this path should not contain project-base. Basically, all of this config files need to be tested completely.